charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -21,6 +21,7 @@ data:
     http-server.http.enabled=false
     http-server.https.enabled=true
     http-server.https.port=8080
+    http-server.https.keystore.key=notnull
```

Comment:
I wonder if `changeit` is more appropriate for the time being as I believe that's the default keystore password for the java trust store.

Reply:
done in 639a758f